src/native/corehost/hostpolicy/deps_format.h

```diff
@@ -26,11 +26,18 @@ class deps_json_t
 public:
     typedef str_to_vector_map_t rid_fallback_graph_t;
 
-    deps_json_t(bool is_framework_dependent, const pal::string_t& deps_path, const rid_fallback_graph_t* graph)
+    struct rid_resolution_options_t
+    {
+        bool use_fallback_graph;
+        const deps_json_t::rid_fallback_graph_t* rid_fallback_graph;
```

Comment:
I find it confusing that we have two places which store RID graph. This structure and then the `m_rid_fallback_graph` field. And sometimes they're the same and sometimes they may not be.
Definitely worth a comment, maybe rename some of these as well.

Reply:
Added comments to this struct and refactored things so that we no longer have the member variable. There's still some weirdness, but hopefully it is a bit clearer now.